src/cfnlint/registry.py

```diff
@@ -0,0 +1,87 @@
+"""
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+
+import json
+import logging
+import os
+import platform
+import subprocess
+from cfnlint.template import Template
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Registry(object):
```

Comment:
np: since this class does both managing resource schemas and interact with CFN registry. What about renaming it `SchemaManager` or some similar names?

Reply:
Good suggestion